src/connector.test.ts

```diff
@@ -13,30 +15,16 @@ async function getConnect(moduleName: string) {
   return connect;
 }
 
-async function smokeDB(db: Database) {
-  const ran = Math.random().toString();
-  for (let i = 0; i < 10; i++) {
-    await db.put({ _id: `key${i}:${ran}`, hello: `world${i}` });
-  }
-  for (let i = 0; i < 10; i++) {
-    expect(await db.get<{ hello: string }>(`key${i}:${ran}`)).toEqual({
-      _id: `key${i}:${ran}`,
-      hello: `world${i}`,
-    });
-  }
-  const docs = await db.allDocs();
-  expect(docs.rows.length).toBeGreaterThan(9);
-  return docs.rows.map((row) => row.value);
-}
-
+// MUST go if superthis is there
 interface ExtendedGateway extends bs.Gateway {
-  logger: { _attributes: { module: string; url?: string } };
+  // logger: { _attributes: { module: string; url?: string } };
   headerSize: number;
   fidLength: number;
   handleByteHeads: (meta: Uint8Array) => Promise<bs.VoidResult>;
 }
 
-interface ExtendedStore {
+// MUST go if superthis is there
```

Comment:
what does this mean?

Reply:
see in core -> all....test.ts it's not more necessary --- all data you need for these tests is now direct accessible.